app/presenters/menu/default_menu.rb

```diff
@@ -94,7 +95,7 @@ def infrastructure_menu_section
           Menu::Item.new('ems_cluster',       N_("Clusters"),          'ems_cluster',                {:feature => 'ems_cluster_show_list'},           '/ems_cluster/show_list'),
           Menu::Item.new('host',              N_("Hosts"),             'host',                       {:feature => 'host_show_list'},                  '/host/show_list'),
           Menu::Item.new('vm_infra',          N_('Virtual Machines'),  'vm_infra_explorer',          {:feature => 'vm_infra_explorer', :any => true}, '/vm_infra/explorer'),
-          Menu::Item.new('resource_pool',     N_('Resource Pools'),    'resource_pool',              {:feature => 'resource_pool_show_list'},         '/resource_pool/show_list'),
+          Menu::Item.new('resource_pool_infra', N_('Resource Pools'),  'resource_pool_infra',        {:feature => 'resource_pool_show_list'},         '/resource_pool_infra/show_list'),
```

Comment:
I noticed the URL is changing.  I wonder if this affects shortcuts?  I can't remember if resource pools was a page that was a shortcut candidate.

Reply:
Should separate entries be added for 'resource_pool_cloud' and 'resource_pool_infra' in the [miq_shortcuts.yaml](https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_shortcuts.yml#L157C2-L161C17) file to accommodate the URL changes?